src/vue/common/CollectionRenderer.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="pagination">
+    <button
+      class="pagination__more-button"
+      @click="loadData(false)">
+      {{ 'common.button-label-more' | globalize }}
+    </button>
+  </div>
+</template>
+
+<script>
+/**
+ * This component is responsible for downloading data in
+ * which there is a pagination. Component return loaded data with action.
+ *
+ * @props {Function} firstPageLoader - function for downloading first page of
+ * data(with filters)
+ *
+ * Example:
+ *
+ * <collection-renderer
+ * :first-page-loader="function() {
+ * Sdk.horizon.transaction.getAll(filters)}"
+ * @load-first-page="handlerData"/>
+ * @load-next-page="handlerNextData"
+ * />
+ *
+ * You can pass a computed property that tracks filter changes and returned new
+ * function.
+**/
```

Comment:
remove `You can pass a computed property that tracks filter changes and returned new function.`